pkg/report/report.go

```diff
@@ -67,6 +68,7 @@ type Stats struct {
 	ErrorDist  map[string]int
 	Lats       []float64
 	TimeSeries TimeSeries
+	Weight     float64
```

Comment:
can this be part of `Result` like the `Ops` field was since different operations may have different weights?